packages/stark-core/src/common/bootstrap/abstract-stark-main.ts

```diff
@@ -137,7 +143,6 @@ We need great software developers like you! https://jobs.nbb.be
 		// written like this because otherwise "this" will not be captured :)
 		if (this.environment.production) {
 			console.log("Customizing configuration for production!");
-			enableProdMode();
 			disableDebugTools();
```

Comment:
Maybe could we move "disableDebugTools" with enableProduction and remove the if condition ? 😊

Reply:
Mmm I don't think we can get rid of the if condition, because we need to enable/disable the debug tools depending on the environment. So at least we should have the `if (!this.environment.production)`.

So I think we can just simply leave it like it is to see what we do different in dev and prod